### PR TITLE
vbfh_5f_NLO_65 cards are added

### PR DESCRIPTION
**Pull request by gourangakole**

### CODE REVIEW

**Comment by govoni** [comment]
please let us know when you have a stable version

**Comment by gourangakole** [comment]
Now I have added all cards for four processes: vbfh, ggh, tth and vh for mass 65 GeV and ready to merge.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh012j_5f_NLO_FXFX_65/ggh012j_5f_NLO_FXFX_65_customizecards.dat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+#set param_card mass 6 173.0
+#set param_card yukawa 6 173.0
+set param_card mass 25 60.0
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh012j_5f_NLO_FXFX_65/ggh012j_5f_NLO_FXFX_65_customizecards.dat]: shuoldn't this be 65?

Comment (gourangakole) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh012j_5f_NLO_FXFX_65/ggh012j_5f_NLO_FXFX_65_customizecards.dat]: Thanks for spotting this..fixed now.

**Pull request merged**